client/login/magic-login/emailed-login-link-expired.jsx

```diff
@@ -12,14 +12,23 @@ import addQueryArgs from 'lib/route/add-query-args';
 import config from 'config';
 import EmptyContent from 'components/empty-content';
 import RedirectWhenLoggedIn from 'components/redirect-when-logged-in';
+import { recordPageView } from 'state/analytics/actions';
 
 const lostPasswordURL = addQueryArgs( {
 	action: 'lostpassword',
 }, config( 'login_url' ) );
 
 class EmailedLoginLinkExpired extends React.Component {
+	static propTypes = {
+		recordPageView: PropTypes.func.isRequired,
+		translate: PropTypes.func.isRequired,
+	};
+
 	render() {
-		const { translate } = this.props;
+		const {
+			translate,
+		} = this.props;
```

Comment:
This is absolutely nitpicking but I don't see a reason for breaking that destructuring into three lines.